Add adjusted template for invoices

lukaspanni opened the pull request:
Added an additional template for invoices based on the letter template.
Could also be combined into one, but I'm unsure on what the better solution would be.

benedictdudel commented:
Hi 👋 

thank you for your contribution ❤️ I really love the idea of a template for invoices. I will come back to you in the next few days!

benedictdudel commented:
Over the last couple of days, I had done a refactoring of the template. The problem with the template before was that a lot of settings were missing which the official default template of pandoc included. The template is now based on the official and has just little tweaks for the scrlttr2 document class.

The project now has a dedicated `examples` directory where multiple letter variants can be published. I already started with an invoice and if you like you could add your footer there. You should use a custom `.lco` file for the footer. You can find more information [here](https://github.com/benedictdudel/pandoc-letter-din5008#1-letter-class-options-lco).

lukaspanni commented:
The new options look great!
I am going to look into providing an update so that it works for my workflow.
Will probably postpone to next year.

lukaspanni committed:
Update to include invoice-footer

lukaspanni commented:
I updated the invoice-footer and introduced a variable to toggle it.
I believe that this makes using it easier for users since they don't have to worry about having the .lco file in the right directory.
If you don't agree I will make a .lco available
